src/core/builtins/builtins_files.ml

```diff
@@ -112,6 +112,10 @@ let _ =
   Lang.add_builtin ~base:file "mkdir" ~category:`File
     ~descr:"Create a directory."
     [
+      ( "parents",
```

Comment:
I think `recursive` is a more common term, that's also the name of the command line option.

Reply:
Hmm no my bad I got confused with `rm`.